Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -116,10 +116,10 @@ They have to be bind-mounted as volumes in the CephFS and RBD plugin pods.
 
 If you install Rook with Helm, uncomment these example settings in `values.yaml`:
 
-- `csiCephFSPluginVolume`
-- `csiCephFSPluginVolumeMount`
-- `csiRBDPluginVolume`
-- `csiRBDPluginVolumeMount`
+- `csi.csiCephFSPluginVolume`
```

Comment:
IMO the csi prefix isn't needed in the doc, they will be able to search for them easy enough without it.

Reply:
However, in my experience, I just did what the docs say and went to my own `values.yaml` and copy-pasted whatever was found there.

I only noticed the need for `csi` prefix when I saw the prefix in the docs. So I guess that having the prefix here will make it more evident for other dummies like me.